e2e/test/scenarios/dashboard-filters-2/shared/dashboard-filters-query-stages.ts

```diff
@@ -621,20 +621,17 @@ export function setup2ndStageAggregationFilter() {
     getPopoverList().scrollTo("bottom");
     getPopoverItem("Count", 1).click();
   });
-  dismissToast();
```

Comment:
The toast previously appeared because of over-matching. See my comment about `getMappingOptionByTarget`